shared-bindings/audiofilters/Filter.c

```diff
@@ -68,9 +69,9 @@
 //|               time.sleep(5)"""
 //|         ...
 static mp_obj_t audiofilters_filter_make_new(const mp_obj_type_t *type, size_t n_args, size_t n_kw, const mp_obj_t *all_args) {
-    enum { ARG_filter, ARG_mix, ARG_buffer_size, ARG_sample_rate, ARG_bits_per_sample, ARG_samples_signed, ARG_channel_count, };
+    enum { ARG_filters, ARG_mix, ARG_buffer_size, ARG_sample_rate, ARG_bits_per_sample, ARG_samples_signed, ARG_channel_count, };
```

Comment:
@dhalbert can we ignore this backward-incompatible change? audiofilters "filter" argument was in 9.2.0 but would not be in 9.2.1.

Reply:
It may take some work to implement, but would the ideal scenario be `filter: Optional[synthio.Biquad|Tuple[synthio.Biquad]]`? It would retain backwards compatibility and reduce confusion if the user only plans on using a single biquad.